CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ set(IGN_TRANSPORT_VER ${ignition-transport8_VERSION_MAJOR})
 
 #--------------------------------------
 # Find ignition-rendering
+# TODO(chapulina) Remove this once ign-rendering exports the variable
+# https://github.com/ignitionrobotics/ign-rendering/issues/360
+if ((NOT ${CMAKE_VERSION} VERSION_LESS 3.11) AND (NOT OpenGL_GL_PREFERENCE))
+  set(OpenGL_GL_PREFERENCE "GLVND")
+endif()
```

Comment:
This workaround fixed the warning locally for me (on Focal) but doesn't fix CI?

Reply:
Ahh that's coming from the examples. Let me try a different fix

Reply:
Trying a different fix in 1f7164b . That's something we should have done a while back anyway. Ignition GUI shouldn't have a hard-dependency on any specific rendering engine, so Ogre2 users don't need Ogre1 installed for example.

We've done the same change on Ignition Gazebo not to depend on any specific rendering or physics engine, but I'm having a hard time finding tickets to cross-link to.

Reply:
Ok, it looks like removing the Ogre component worked on CI too